trashtalk/views/cleanups.py

```diff
@@ -56,15 +56,15 @@ def get(cleanup_id):
     else:
         bool_participated = False #Default to anonymous users not being participants
 
+    map = current_app.config['GOOGLE_MAPS_ENDPOINT'].format(cleanups.gmap_query, current_app.config['GOOGLE_MAPS_KEY'])
```

Comment:
Recommend set the value for gmap directly or:
```
query_result = current_app.config['GOOGLE_MAPS_ENDPOINT'] + cleanups.gmap_query

# OR
query_result = "{0}{1}".format(current_app.config['GOOGLE_MAPS_ENDPOINT'], cleanups.gmap_query)

# OR 
gmap=current_app.config['GOOGLE_MAPS_ENDPOINT'] + cleanups.gmap_query
```

Reply:
###### This comment is in another pull review. I copied it here. 
```
This formatting has caused me issues for months. 
- Google wants the key at the end of the URL 
- If the key is at the end, the coordinates need to be defined in the center
- If the coordinates are defined in the center, they need to be defined using formatting
- All variables defined using formatting must be defined at the same time
- The API Key can't be defined until the coordinates are defined
😩 
```

Reply:
I checked out the issue. I wasn't able to reproduce this error. I think it's related to the combination of changes here so I'll check it out. I'll also revert the gmaps constant to its original. Settings should not include logic. For example, the query should be handled in the module in which it's used not formatted to the settings.

Reply:
Also settings should be settings. Queries are handled at the module level. That means keeping `GOOGLE_MAPS_ENDPOINT` as strictly the endpoint with its api key in the settings file, and then using the endpoint with a query in the modules. Suggestions above for example.

Reply:
I'll let you fight that battle.